toxcore/tox.h

```diff
@@ -4620,7 +4620,7 @@ bool tox_group_send_custom_private_packet(const Tox *tox, uint32_t group_number,
  * @param peer_id The ID of the peer who sent the message.
  * @param type The type of message (normal, action, ...).
  * @param message The message data.
- * @param pseudo_msg_id a pseudo message id that clients can use to uniquely identify this group message.
+ * @param pseudo_msg_id A pseudo message id that clients can use to uniquely identify this group message.
```

Comment:
This param should also be renamed: pseudo_msg_id -> message_id. The fact that it's a "pseudo" message ID is unnecessary internal details.